app/services/dunning_campaigns/process_attempt_service.rb

```diff
@@ -16,7 +16,6 @@ def call
       return result unless applicable_dunning_campaign?
       return result unless dunning_campaign_threshold_reached?
       return result unless days_between_attempts_passed?
-      return result if customer.dunning_campaign_completed?
```

Comment:
we still need to check if the max attempts has been reached here...

please revert the changes in this commit:

https://github.com/getlago/lago-api/pull/2889/commits/0d72019ad0625f36c332bb53f167e7c77f57a5c1